website/src/components/EventSlider/index.tsx

```diff
@@ -66,10 +66,10 @@ const EventSlider: React.FC<EventsProps> = ({
             const dateB: any = new Date(eventA?.date);
             return sortOrder === "asc" ? dateB - dateA : dateB + dateA;
           })
-        : eventsData;
-      const allEvents = [...eventsData, ...recurringEvents];
-      setFilteredEvents([...allEvents]);    
+        : eventsData;   
     }
+    const allEvents = [...eventsData, ...recurringEvents];
+    setFilteredEvents([...allEvents]); 
```

Comment:
```suggestion
    setFilteredEvents(allEvents); 
```

Reply:
since we are already using rest operator on one line above :)

Reply:
That throws a type incompatible error.

Reply:
Array are reference types. So we should we spread here.